### PR TITLE
EDU-112-86: Section override (Part I - Edited files)

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
#### Types of changes
- [x] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

- `navigation.json` updated: [PR 646](https://github.com/vtexdocs/devportal/pull/646) 
- Jira task: [EDU-11286](https://vtex-dev.atlassian.net/browse/EDU-11286) 

[EDU-11286]: https://vtex-dev.atlassian.net/browse/EDU-11286?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/faststore/docs/customization/building-sections/overrides/component-props.mdx [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+To integrate the section into the [Headless CMS](https://developers.vtex.com/docs/guides/faststore/headless-cms-overview), follow the steps below:
+
+1. Create a folder named `cms` in the FastStore root directory.
+
+2. Inside `cms`, create the `faststore` folder.
+
+3. Within the `cms/faststore` folder, create the `sections.json` file.
+
+4. Add the schema of the new section to the `sections.json` file, named `ProductDetailsWithCustomButton`, to the `sections.json` file:
```
Comment (carolinamenezes) [inline docs/faststore/docs/customization/building-sections/overrides/component-props.mdx:99]:
```suggestion
1. Create a folder named `cms` in the FastStore root directory.
2. Inside `cms`, create the `faststore` folder.
3. Within the `cms/faststore` folder, create the `sections.json` file.
4. Add the schema of the new section to the `sections.json` file, named `ProductDetailsWithCustomButton`, to the `sections.json` file:
```

File: docs/faststore/docs/customization/building-sections/overrides/component-props.mdx [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+    yarn cms-sync
+    ```
+
+7. Access the VTEX Admin and proceed to **Storefront > Headless CMS**.
+
+8. Click on the page you desire to add the new section.
+
+9. Click the add button (`+`) and add the new section and complete values from the sections fields.
+
+10. Click `Save`.
```
Comment (carolinamenezes) [inline docs/faststore/docs/customization/building-sections/overrides/component-props.mdx:255]:
```suggestion
5. Open a new terminal window and log in to your vtex account by running `vtex login {accountName}`.
6. run the following command to sync your changes with the Headless CMS:

    ```bash
    yarn cms-sync
    ```

7. Access the VTEX Admin and proceed to **Storefront > Headless CMS**.
8. Click on the page you desire to add the new section.
9. Click the add button (`+`) and add the new section and complete values from the sections fields.
10. Click `Save`.
```

File: docs/faststore/docs/customization/building-sections/overview.mdx [hunk 3]
```diff
@@ -1,62 +1,34 @@
...
-- **Styling custom components**
-
-When styling custom components created for overrides, follow the guidelines provided in [Customizing a new component when using override](https://developers.vtex.com/docs/guides/faststore/using-themes-components#customizing-a-new-component-when-using-override). These guidelines will help you maintain consistency and ensure your custom components integrate seamlessly with the rest of the FastStore ecosystem.
-
-- **Organizing files and folders**
+### API extension
 
-For effective organization, follow the steps in the [setting up the overrides folder](https://developers.vtex.com/docs/guides/faststore/building-sections-overriding-components-and-props#step-by-step%23step-1-setting-up-the-overrides-folder) section. This will help you organize, create, and name the overrides and ensure they are properly integrated into your project.
+Add new data to existing queries on the FastStore API by extending it. Refer to the [API extension guide](https://developers.vtex.com/docs/guides/faststore/api-extensions-overview) for more
+information.
```
Comment (carolinamenezes) [inline docs/faststore/docs/customization/building-sections/overview.mdx:34]:
```suggestion
Add new data to existing queries on the FastStore API by extending it. Refer to the [API extension guide](https://developers.vtex.com/docs/guides/faststore/api-extensions-overview) for more information.
```

**Review by carolinamenezes — APPROVED** [review]

File: docs/faststore/docs/customization/building-sections/overrides/component-props.mdx [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+
+## Before you begin
+
+Make sure your `@faststore/core` package has the 3.x version or above. If you need to update
+it, refer to this [release note](https://github.com/vtex/faststore/releases/tag/v3.0.0).
+
+## Step by step
+### Step 1 - Setting up the component file
+
+1. After choosing a native section to be customized from the [list of available native sections](/LINK), open your FastStore project in any code editor of your preference.
```
Comment (Mariana-Caetano) [inline docs/faststore/docs/customization/building-sections/overrides/component-props.mdx:22]:
```suggestion
1. After choosing a native section to be customized from the [list of available native sections](https://developers.vtex.com/docs/guides/faststore/building-sections-list-of-native-sections), open your FastStore project in any code editor of your preference.
```

File: docs/faststore/docs/customization/building-sections/overrides/section-styles.mdx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+To customize the styles of a native section, you can use the `className` property, which works similarly to [React's className](https://react.dev/reference/react-dom/components/common#applying-css-styles).
+
+## Before you begin
+
+Before delving into this guide, make sure you have already overridden a component, as presented in the [getOverriddenSection function] guide.
+
+## Step by step
+
+
+Let’s use the example provided in the [getOverriddenSection function](LINK) guide,where the `Simple Alert` styles are being overridden via the `className`:
```
Comment (Mariana-Caetano) [inline docs/faststore/docs/customization/building-sections/overrides/section-styles.mdx:14]:
```suggestion
Let’s use the example provided in the [getOverriddenSection function](https://developers.vtex.com/docs/guides/faststore/overrides-getoverriddensection-function) guide, where the `Simple Alert` styles are being overridden via the `className`:
```

File: docs/faststore/docs/customization/building-sections/overrides/getoverriddensection-function.mdx [hunk 6]
```diff
@@ -0,0 +1,63 @@
+---
+title: '`getOverriddenSection` function'
```
Comment (Mariana-Caetano) [inline docs/faststore/docs/customization/building-sections/overrides/getoverriddensection-function.mdx:2]:
```suggestion
title: 'getOverriddenSection function'
```

**Pull request merged**